pkg/scanner/secrets.go

```diff
@@ -372,6 +372,13 @@ func GetSecretRegexes() []Secret {
 			[]string{},
 			"?",
 		},
+		{
+			"Alibaba OSS Bucket",
+			"Alibaba OSS Bucket",
+			`(?:[a-zA-Z0-9-\.\_]+\.oss-[a-zA-Z0-9-\.\_]+\.aliyuncs\.com|oss\:\/\/[a-zA-Z0-9-\.\_]+|oss-[a-zA-Z0-9-\.\_]+\.aliyuncs\.com/[a-zA-Z0-9-\.\_]+)`,
```

Comment:
- Linter is complaining about line too long, see other regexes, e.g. AWS bucket one
- The regex `oss\:\/\/[a-zA-Z0-9-\.\_]+` can be moved to AWS since it recognizes oss protocol, which is for AWS S3 compatible target

Reply:
Thanks for the changes. 
We can fix those errors using https://pkg.go.dev/fmt#example-Sprintf, should fix them.

Let me know if you need any assistance :)